pilot/pkg/model/context.go

```diff
@@ -74,6 +74,11 @@ type Proxy struct {
 	// co-located service instances. Example: "10.60.1.6"
 	IPAddress string
```

Comment:
Convert this into an array and fix all callsites..

Reply:
Makes sense, thanks for the suggestion, I'm working on it.

Reply:
After looking into the impacted codes, I think we'd better distinguish the management IP from other IPs.  Unlike the service IPs, we must ensure that management IP is reachable by the control plane, it can't be any of these local IPs reported by proxy.

Reply:
OK, we can just pass all the IPs to the service registry controllers and let them decide which IP should be used to create management listeners, this approach is more transparent and don't need any particular configuration on the proxy side.